### ISSUE TITLE
Training code for SEED-LLaMA

### ISSUE THREAD

**Issue by shubhamgarg21** [AILab-CVC/SEED#12]
Hi,
I am wondering if at any point the training code for SEED-LLaMA will be made available?

**Comment by geyuying** [comment]
We have released the training code of SEED-LLaMa, including SEED tokenizer, Multimodal LLM pretraining and instruction tuning. Our Multimodal LLM training codebase supports 1. large-scale multi-node training with deepspeed 2. highly-efficient multiple training datapipes.
